ginga/misc/plugins/Cuts.py

```diff
@@ -227,6 +249,22 @@ def build_gui(self, container):
         self.select_cut(self.cutstag)
         self.gui_up = True
 
+    def build_axes(self, vbox):
+        fr = Widgets.Frame("Axes")
+        hbox = Widgets.HBox()
+        hbox.set_border_width(4)
+        hbox.set_spacing(1)
+
```

Comment:
Since the axes checkboxes are only used for the slit, and only useful if there is a 3D cube, I recommend moving the checkbox array into the bottom of the slit tab instead of the main window.  I suggest building the hbox holder in `build_gui` and just keeping a handle to it, so that you can just clear it and repopulate it with new checkboxes when a new image arrives.